vdr/api/v2/api.go

```diff
@@ -104,19 +105,48 @@ func (a *Wrapper) ListDIDs(ctx context.Context, _ ListDIDsRequestObject) (ListDI
 	return ListDIDs200JSONResponse(result), nil
 }
 
-func (w Wrapper) AddService(ctx context.Context, request AddServiceRequestObject) (AddServiceResponseObject, error) {
-	//TODO implement me
-	panic("implement me")
+func (w Wrapper) CreateService(ctx context.Context, request CreateServiceRequestObject) (CreateServiceResponseObject, error) {
+	targetDID, err := did.ParseDID(request.Did)
+	if err != nil {
+		return nil, err
+	}
+	createdService, err := w.VDR.CreateService(ctx, *targetDID, *request.Body)
+	if err != nil {
+		return nil, err
+	}
+	return CreateService200JSONResponse(*createdService), nil
 }
 
 func (w Wrapper) DeleteService(ctx context.Context, request DeleteServiceRequestObject) (DeleteServiceResponseObject, error) {
-	//TODO implement me
-	panic("implement me")
+	targetDID, err := did.ParseDID(request.Did)
+	if err != nil {
+		return nil, err
+	}
+	serviceID, err := ssi.ParseURI(request.ServiceId)
+	if err != nil {
+		return nil, err
+	}
+	err = w.VDR.DeleteService(ctx, *targetDID, *serviceID)
```

Comment:
Can result in an `didweb.errServiceNotFound` which is not wrapped by the VDR/management which then will result in a 500 error, should be in the 400 range. Same goes for other errors and other services. (including unique key violations, etc)

Reply:
Now mapping these errors to `management.ErrInvalidService`, which is then mapped to HTTP 400